rest_framework_simplejwt/authentication.py

```diff
@@ -108,8 +111,8 @@ def get_user(self, validated_token):
             raise InvalidToken(_('Token contained no recognizable user identification'))
 
         try:
-            user = User.objects.get(**{api_settings.USER_ID_FIELD: user_id})
-        except User.DoesNotExist:
+            user = self.user_model.objects.get(**{api_settings.USER_ID_FIELD: user_id})
+        except self.user_model.DoesNotExist:
```

Comment:
I'd much rather see something above the class: `User = get_user_model()`  than see what's being done here.